api.go

```diff
@@ -77,16 +78,28 @@ func (e *ErrorResponse) Temporary() bool {
 }
 
 func (api *API) request(endpoint string, method string, params map[string]interface{}, body *bytes.Buffer) (result *bytes.Buffer, status int, err error) {
+	bucketLimit, err := strconv.Atoi(os.Getenv("BUCKET_LIMIT"))
```

Comment:
Good point, have added a Config struct to house them. Can you have another look when you are free? Thank you!